pkg/execution/state/redis_state/redis_state.go

```diff
@@ -277,14 +267,14 @@ func (m mgr) New(ctx context.Context, input state.Input) (state.State, error) {
 		return nil, err
 	}
 
-	status, err := scripts["new"].Exec(
+	status, err := retriableScripts["new"].Exec(
```

Comment:
This function shows how we supply sharding-enabled keys to a retriable script on a cluster client. Since New() is placed on a sharded manager, we could not access unsharded keys or clients in here.